src/components/header.tsx

```diff
@@ -102,12 +102,15 @@ const Header: React.FC<Props> = ({ className, withMenu, onMenuClick }) => {
           >
             Docs
           </HeaderLink>
-          <Only
-            from="sm"
-            as={HeaderLink}
-            href="https://github.com/mswjs/examples"
-          >
-            Examples
+          <Only from="sm">
```

Comment:
Could you please set this `Only` to have the `flex` prop?
That way it will stretch vertically, and the active link's border will align with the header's border. Thanks!

```diff
- <Only from="sm">
+ <Only from="sm flex>
```